embedded-graphics/src/style/mod.rs

```diff
@@ -6,4 +6,106 @@ mod text_style;
 
 pub use primitive_style::{PrimitiveStyle, PrimitiveStyleBuilder};
 pub use styled::Styled;
-pub use text_style::TextStyle;
+pub use text_style::{TextStyle, TextStyleBuilder};
+
+/// Create a [`PrimitiveStyle`]
+///
+/// Any properties not specified in the macro call will use the values provided by
+/// `PrimitiveStyle::default()`.
+///
+/// ```rust
+/// use embedded_graphics::{
+///     pixelcolor::{Rgb565, RgbColor},
+///     primitive_style,
+/// };
+///
+/// let style = primitive_style!(fill_color = Rgb565::RED);
+/// ```
+///
+/// [`PrimitiveStyle`]: ./style/struct.PrimitiveStyle.html
+
+#[macro_export]
+macro_rules! primitive_style {
+    ($($style_key:ident = $style_value:expr ),* $(,)?) => {{
+        #[allow(unused_mut)]
+        let mut builder = $crate::style::PrimitiveStyleBuilder::new();
+
+        $( builder.$style_key($style_value); )*
+
+        builder.build()
+
+    }};
+}
+
+/// Create a [`TextStyle`]
+///
+/// All properties on [`TextStyle`] are supported. At least `font` is required, and must be the
+/// first property passed to the macro.
+///
+/// ## Examples
+///
+/// ### Create a default text style
+///
+/// This example uses [`Font8x16`] and the [`Rgb565`] colour space to create a default text style.
```

Comment:
Sorry for this, but:
```suggestion
/// This example uses [`Font8x16`] and the [`Rgb565`] color space to create a default text style.
```
(This also applies to the British colours below)

Reply:
:(((((

I did a find and replace through the repo and changed every occurrence I could find, including some test image file names.